StyleGuide.md

```diff
@@ -1021,35 +1021,18 @@ The syntax is a little hard to read inline, but based on some experimenting it a
 Formatted for easier reading, the template code looks like:
 
 ```
-{% if page.name == "cost-based-optimizer.md" %}  
-Locality-optimized search  
-{% else %}  
-[Locality-optimized search](cost-based-optimizer.html#locality-optimized-search-in-multi-region-clusters)  
-{% endif %}  
+{% if page.name == "cost-based-optimizer.md" %}
+Locality-optimized search
+{% else %}
+[Locality-optimized search](cost-based-optimizer.html#locality-optimized-search-in-multi-region-clusters)
+{% endif %}
 ```
 
-##### Different content depending on CockroachCloud or Self-Hosted
-
-Sometimes, you must use an include file on both a CockroachCloud and a Self-Hosted page. This requires that the links be resolved differently depending on the page that the include file is being referenced from.
-
-For instructions showing how to use relative links that differ depending on the page type, see [Use relative links to Cloud and Core topics in _include files](https://cockroachlabs.atlassian.net/wiki/x/woBZl).
-
-_Note_: In the previous link, Cloud = Dedicated and Core = Self-Hosted.
-
 <a name="remote-includes"></a>
 
 ##### Remote includes
 
-Sometimes, you need to include files that are maintained in other places than the `cockroachdb/docs` repo but referenced in our docs. The `remote_include` tag is used for this. We most often use this tag for:
-
-- SQL diagrams, which are maintained in the `cockroachdb/cockroach` repo
-- Code samples, which are maintained in various repos
-
-For SQL diagrams, you remotely include the entire (HTML) file as follows:
-
-```
-{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-{{ page.version.version | replace: "v", "" }}/grammar_svg/show_databases.html %}
-```
+Sometimes, you need to include files that are maintained in other places than the `cockroachdb/docs` repo but referenced in our docs. The `remote_include` tag is used for this. We most often use this tag for code samples, which are maintained in various repos.
```

Comment:
```suggestion
Sometimes, you need to include files that are maintained in other places than the `cockroachdb/docs` repo but referenced in our docs. The `remote_include` tag is used for this. We most often use this tag for code samples, which are maintained in various repos controlled by Cockroach Labs.
```

Nit: do you find value in this addition?

Reply:
I don't think it's aways true. One of the examples `remote_include https://raw.githubusercontent.com/jeffgbutler/mybatis-cockroach-demo/master/src/main/java/com/example/cockroachdemo/CockroachDemoApplication.java` points to this repo: `https://github.com/jeffgbutler/mybatis-cockroach-demo`

Reply:
Fair. I'll scrap the suggestion. LGTM!